terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/codelist/08_02_form.jsp

```diff
@@ -17,17 +19,19 @@
         <td><form:input path="item2" /></td>
       </tr>
       <tr>
-        <td><input id="btn1" class="mainbtn" style="width: 200px;" type="submit" value="Confirm"
-          name="existInCheckCharacter" />
+        <td><input id="btn1" class="mainbtn" style="width: 200px;"
+          type="submit" value="Confirm" name="existInCheckCharacter" />
         <td>
         <td></td>
       </tr>
     </table>
   </form:form>
-  <form:form action="${pageContext.request.contextPath}/codelist" method="get">
+  <form:form action="${pageContext.request.contextPath}/codelist"
+    method="get">
     <table>
       <tr>
-        <td><input id="btnback" class="mainbtn" style="width: 200px;" type="submit" value="Back to menu" /></td>
+        <td><input id="btnback" class="mainbtn"
+          style="width: 200px;" type="submit" value="Back to menu" /></td>

```

Comment:
- [ ] Is not it hard to see? I think that it is better to make one line long enough. What do you think?